auth/auth.go

```diff
@@ -9,7 +9,7 @@ import (
 
 	"crypto/rand"
 
-	"github.com/gophish/gophish/models"
+	"github.com/jli53/gophish/models"
```

Comment:
All these dependencies will need to be reverted to gophish/gophish - I know it's a pain to work with when forking.